test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -64,11 +64,16 @@ Describe "Test-Connection" -tags "CI" {
             { $result = Test-Connection "fakeHost" -Count 1 -Quiet -ErrorAction Stop } |
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.Quiet works
```powershell
Expected $true, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.164.3/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 272
272:             $result | Should -BeTrue
```